README.md

```diff
@@ -432,7 +392,7 @@ class RethinkDBFSBucket {
   /**
    * Find and return the files table documents that match @filter.
    */
-  Iterable find(Document filter, RethinkDBFSFindOptions options=null);
+  Iterable find(Document filter, RethinkDBFSFindOptions=null);
 
```

Comment:
Nah, not really useful here, actually. I kinda feel it's okay to loosen the strictness of the API methods and arguments. They had one object `RethinkDBFSDownloadByNameOptions` that inherited from  `RethinkDBFSDownloadOptions`. `*ByNameOptions` defined a single property called *revision* just to specify the revision number. So, instead of a kluge way of fumbling with different type names, just made it a parameter to the method:

```
public byte[] DownloadBytesByName(string fileName, int revision = -1, DownloadOptions options = null)
```
and `DownloadOptions` becomes a single type over all the download methods. Has a much better surface semantics in .NET. I guess they like to over-normalize their parameters / methods. :palm_tree: